nrf_802154/doc/CHANGELOG.rst

```diff
@@ -31,6 +31,7 @@ Other changes
 * Removed the ``NRF_802154_TX_STARTED_NOTIFY_ENABLED`` config option. (KRKNWK-16364)
 * The total times measurement feature is turned off. (KRKNWK-16189)
 * Removed the ``NRF_802154_TOTAL_TIMES_MEASUREMENT_ENABLED`` config option and support for the total times measurement feature. (KRKNWK-16374)
+* CSL Phase is calculated assuming that provided CSL anchor time points to a time where the first bit of MAC header of the frame received from a peer happens. (KRKNWK-16647)
```

Comment:
Not sure about what "points to a time" and "happens" mean in this context.
Could you provide some context?

I think these 2 terms need to be changed.

Reply:
I'd suggest replacing `points to` with `specifies` and `happens` with `starts`. Is such wording better?

Reply:
Applied.